src/extensions/default/bramble/main.js

```diff
@@ -36,24 +36,12 @@ define(function (require, exports, module) {
     ExtensionUtils.loadStyleSheet(module, "stylesheets/sidebarTheme.css");
 
     function parseData(data) {
-        var dataReceived = data;
-
         try {
-            data = dataReceived || null;
-            data = JSON.parse(data);
-            data = data || {};
+            data = JSON.parse(data || null);
+            return data || {};
         } catch(err) {
-            // Quick fix: Ignore the 'process-tick' message being sent
-            if(dataReceived === "process-tick") {
```

Comment:
It's not that we don't need it, but that we don't need to special-case this--we always `return false` since it just means that we got a message we don't care about in this context.  It's common for this to happen now, since we have multiple windows pumping messages into this one (host app and preview iframe).